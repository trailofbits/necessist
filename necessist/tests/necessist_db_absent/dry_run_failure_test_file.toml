args = [
    "--no-sqlite",
    "--root=examples/dry_run_failure",
    "examples/dry_run_failure/tests/b.rs",
]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
