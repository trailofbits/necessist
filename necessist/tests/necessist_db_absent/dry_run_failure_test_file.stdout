1 candidates in 1 test file
examples/dry_run_failure/tests/b.rs: dry running
examples/dry_run_failure/tests/b.rs: mutilating
examples/dry_run_failure/tests/b.rs:4:5-4:12: `n += 1;` passed
